src/Action/CaptureAction.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.shop and write us
+ * an email on mikolaj.krol@bitbag.pl.
+ */
+
+declare(strict_types=1);
+
+namespace PayPlug\SyliusPayPlugPlugin\Action;
+
+use PayPlug\SyliusPayPlugPlugin\Action\Api\ApiAwareTrait;
+use PayPlug\SyliusPayPlugPlugin\ApiClient\PayPlugApiClientInterface;
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\ApiAwareInterface;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\GatewayAwareInterface;
+use Payum\Core\GatewayAwareTrait;
+use Payum\Core\Reply\HttpRedirect;
+use Payum\Core\Request\Capture;
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Security\GenericTokenFactoryAwareInterface;
+use Payum\Core\Security\GenericTokenFactoryInterface;
+use Payum\Core\Security\TokenInterface;
+use Payum\Core\Exception\RuntimeException;
+use Psr\Log\LoggerInterface;
+
+final class CaptureAction implements ActionInterface, ApiAwareInterface, GatewayAwareInterface, GenericTokenFactoryAwareInterface
+{
+    use GatewayAwareTrait, ApiAwareTrait;
+
+    /** @var GenericTokenFactoryInterface|null */
+    private $tokenFactory;
+
+    /** @var LoggerInterface */
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function setGenericTokenFactory(GenericTokenFactoryInterface $genericTokenFactory = null): void
```

Comment:
Please, add PHP Doc for every function

Reply:
As we use strict types in every PHP file, the PHPDoc is redundant. That being said, it does not correspond with our recommended Sylius coding standards we described here: https://github.com/BitBagCommerce/BitBagBible.

Reply:
Very well